sys/kern/sysv_shm.c

```diff
@@ -437,9 +437,6 @@ kern_shmat_locked(struct thread *td, int shmid,
 	vm_prot_t prot;
 	vm_size_t size;
 	int cow, error, find_space, i, rv;
-#if __has_feature(capabilities)
-	int reqperm;
-#endif
```

Comment:
FreeBSD style would leave the variable here.

Reply:
style(9) actually says "Declarations may be placed before executable lines at the start of any block.  Calls to complicated functions should be avoided when initializing variables." these days (as of July 2020's 34a8e7368f3f00d4564b0b980871f73929d6b1ce)

Reply:
Right, moving it is good then